tests/styles.spec.js

```diff
@@ -0,0 +1,55 @@
+const assert = require('assert');
+const { execSync } = require('child_process');
+const { mkdirSync, cpSync, readFileSync } = require('fs');
+
+/* eslint-disable */
+let copy;
+if (typeof cpSync === 'undefined') {
+  copy = require('recursive-copy');
+}
+/* eslint-enable */
+
+const { resolve } = require('path');
+const { cli, stdio } = require('./lib/cli');
+const {
+  copyRecursively,
+  createManifestFile,
+  createTemporaryWorkingDirectory,
+  deleteTemporaryWorkingDirectory,
+} = require('./lib/helpers');
+
+const basicStylesPath = resolve(__dirname, 'styles/samples/basic/scss');
+
+describe('Style tasks', () => {
+  describe('checks entire style task', () => {
+    const command = `${cli} styles`;
+    let cwd;
+    // Before test is ran, needs to compile styles into css.
+    before(async () => {
+      cwd = createTemporaryWorkingDirectory();
+      const tmpDirStylesPath = resolve(cwd, 'src/styles');
+      createManifestFile(cwd);
+      execSync('yarn add breakpoint-sass', { cwd, stdio });
+      mkdirSync(resolve(cwd, 'src'));
+      await copyRecursively(basicStylesPath, tmpDirStylesPath);
+      execSync(command, { cwd, stdio });
+    });
+
+    // after test is ran, delete that temporary compiled style.
+    after(() => deleteTemporaryWorkingDirectory(cwd));
```

Comment:
Let’s follow the convention in `init.spec.js` and add an empty line beneath this line:
```suggestion
    after(() => deleteTemporaryWorkingDirectory(cwd));

```